chispa/schema_comparer.py

```diff
@@ -44,33 +50,33 @@ def inner(s1, s2, ignore_nullable, ignore_metadata):
 # deprecate this
```

Comment:
Let's, maybe, add a deprecation warning?

Reply:
There's multiple functions with `# deprecate this` comment. Let's leave solving that for a follow-up PR.